test/state/state.cpp

```diff
@@ -89,6 +89,12 @@ std::variant<int64_t, std::error_code> validate_transaction(const Account& sende
     if (sender_acc.nonce == Account::NonceMax)
         return make_error_code(NONCE_HAS_MAX_VALUE);
 
+    if (sender_acc.nonce < tx.nonce)
```

Comment:
Do we need "to high"/"to low" variants? How about `NONCE_INVALID`?

Reply:
We don't. But blockchain tests required to distinct between them.
`src/BlockchainTestsFiller/ValidBlocks/bcStateTests/TransactionNonceCheckFiller.json`
`src/BlockchainTestsFiller/ValidBlocks/bcStateTests/TransactionNonceCheck2Filler.json`

Reply:
Ok, let it be.